CodeEdit/Features/NavigatorSidebar/NavigatorSidebarToolbarTop.swift

```diff
@@ -73,6 +73,8 @@ struct NavigatorSidebarToolbarTop: View {
             selection = id
         } label: {
             getSafeImage(named: named, accesibilityDescription: title)
+                .resizable()
+                .aspectRatio(contentMode: .fill)
```

Comment:
Wouldn't it be better to use `.fit` instead? `.fill` might lead to cut-off icons if they are particularly wide or tall.

Reply:
You are right. But in this case, I decided to use `.fill` because we mix `nsImage` and `symbol`. They have different spaces inside the image which causes non-uniform size. 

Because we explicitly define hit-test area via `frame` and `contentShape` and we also don't use `clipShape`, I didn't expect any problems with interaction, but visually, they look more consistent.

But if you belive that `.fit` if more appropriate in that case, no problem, I can't change that.

![photo_2023-01-03_19-52-04](https://user-images.githubusercontent.com/18378212/210392973-20f6e0dd-21b4-43d3-91c4-bcde2c9abc9d.jpg)

Reply:
Ah okay I see why you did that! And yeah it should not clip in the current configuration.